client/analysisui/variableslistbox.js

```diff
@@ -169,8 +170,10 @@ const VariablesListBox = function (params) {
         if (itemValue.format.name === 'variable' && item.properties !== undefined)
             allowItem = this._checkIfVariableTypeAllowed(item.properties);
 
-        if ( ! allowItem &&  ! silent)
+        if ( ! allowItem &&  ! silent) {
+            focusLoop.speakMessage(s_('{var} not permitted. Incorrect measure type.', {var: itemValue.toAriaLabel()}));
```

Comment:
incorrect measure type, or incorrect data type?

Reply:
its possibly both, so i'll change to incorrect measure or data type?

Reply:
i don't think we actually restrict on measure type ...

Reply:
when populating variables in the options panel we check measure and data type

Reply:
> i'll change to incorrect measure or data type?

sure